Spaceport Culture

MasterOfGrey opened the pull request:
## Summary
This PR seeks to add various minor events or interactions that can occur in spaceports to give more depth of character to particular areas. (For now, this is human and Hai space.)

The objective is 30+ minor 'events' (missions that are just text) for a standard faction, and probably double that for the republic to break it down by area a bit better.

Any/all suggestions are welcome, this first batch won't meet the goal right now, but I'm not going to hold up what's there for that long.

monticore162 commented on data/human/culture republic.txt:
```diff
@@ -11,28 +11,118 @@
-#mission "cultural random template"
-	landing
+		conversation
+			`As you wander the spaceport with a snack, waiting for your ship to be refuelled, you pass a local souvenir shop and poke your head in to kill a little time. You see a decent looking mug with the text "<source>" printed underneath an attractive, stylised representation of the spaceport you are presently standing in and pick it up to see how it feels. The handle is solid, and not too small for  ...
```
refuelled => refueled

MasterOfGrey replied:
Bloody american spellings. :P  Don't get too hung up on typos at this stage though, it hasn't been subjected to a proper spellcheck yet. (My initial ideas are on the wong computer.)

monticore162 commented on data/human/culture republic.txt:
```diff
@@ -11,28 +11,118 @@
+		conversation
+			`Visiting the Paradise spaceports is always an interesting experience for someone from abroad. Nowhere else is class such an ingrained component of daily life. You catch yourself noticing people who seem to be invisible to most of those going about their business, but they never catch your eye. Perhaps it is because of your background, but you notice them even when they are avoiding notice, an ...
+			`	You haven't noticed children that look poor here in the past, but on this occasion you spot one right before he knocks over a mop and bucket... right in front of a businessman and his family rolling their bags through the spaceport. You've never seen a mother go so deathly pale as the cleaner who scurried to fix things before the man had even drawn breath.`
+			`	A pair of security officers appear almost immediately, before you could intercede one way or the other, as the man and his family begin to complain. One officer provides assurances as the other supervises the cleaner, who quickly hurries her children out of harms way.`
```
is the cleaner the parent of the child or not?

MasterOfGrey replied:
That is the intended implication.

monticore162 commented:
perhaps an earth specific mission where you see an old artillery piece or tank on display

MasterOfGrey commented:
There's already a bunch of Earth specific jobs, so Earth-specific is pretty much the only thing I wasn't considering.
Also, such things are probably in museums, not the spaceport.

MasterOfGrey commented:
It appears some mysterious Korath edits ended up in here when they shouldn't. Nobody panic, we'll clean that up shortly.

Arachi-Lover commented on data/human/culture free worlds.txt:
```diff
@@ -0,0 +1,67 @@
+		conversation
+		`Farm world spaceports tend to be lonely affairs outside of harvesting season, but this one is bustling with festive energy. Rows of colorful tents and throngs of the planet's natives fill the area, as well as a proliferation of craftwork pig-sculptures; some are built from hay, some papier-mache and paint - one even appears to have been carved from butter. Most spaces that aren't taken by a si ...
+		choice
+			`	Carry on with your business. You can always come back next year.`
```
Action choices should be (Like this.)

Arachi-Lover commented on data/human/culture free worlds.txt:
```diff
@@ -0,0 +1,67 @@
+			`	Carry on with your business. You can always come back next year.`
+				defer
+			`	Stop and take in the sights - and maybe a soft pretzel or three.`
+		'Conversation with the locals eventually turns to the matter of what they're celebrating in the first place. As it turns out, the bronzed pig holds the clue. Today is the anniversary of the Great Pig War, a conflict which apparently raged between the Southern Defense Pact and the Syndicate unbeknowst to the larger galaxy. Toasts are raised in honor of nine heroic captains - and to the memory of ...
```
This starts with a ' instead of a `

Also, with the exception of the first line in a conversation, all other lines must be indented once.

W1zrad commented:
Haven't reviewed, but I noticed the files all start with "culture" instead of the faction. Almost all other files follow the format of putting faction first then the contents of the file (ex: hai culture). Was there any specific reason you did in reverse? If not, I'd suggest switching the names around.

Arachi-Lover commented on data/human/culture free worlds.txt:
```diff
@@ -0,0 +1,67 @@
+# "Missions" which add culture events to the spaceport.
+
+mission "South: The Great Pig War"
+	landing
```
Why `landing`? I'd have it minor and still in the spaceport, as it starts off talking about the spaceport.

MasterOfGrey replied:
@Galaucus has some funny ideas. :P

Arachi-Lover replied:
that doesn't answer my question :james:

like I told WarlordMike recently, nobody here reviewing cares about spoilers, spell out what ideas are, get feedback now, see if that changes my mind or if I still think it makes more sense if it's a spaceport-triggered mission

MasterOfGrey replied:
Since you missed it, - I tagged him because all your questions are on his content, I can't answer them.

Arachi-Lover replied:
I see, comment above lead me to believe you knew what the `funny ideas` were.

Arachi-Lover reviewed (changes requested):
It has occurred to me that having dozens of these "useless" spaceport encounters may discourage newer players from checking the spaceport whenever possible, and so they may miss out on missions or whole storylines, because one of these has "taken up" a spaceport where otherwise a mission chain would be offered. Don't see how that could be prevented given there's no "minor+" tag to have something be of lower priority than chains that may already have the "minor" tag

MasterOfGrey commented:
> Haven't reviewed, but I noticed the files all start with "culture" instead of the faction. Almost all other files follow the format of putting faction first then the contents of the file (ex: hai culture). Was there any specific reason you did in reverse? If not, I'd suggest switching the names around.

I'm not entirely convinced the current breakdown will be the final one, so for the moment it just makes them easier to access because they're all grouped together. The file naming will be reviewed closer to completion.

> It has occurred to me that having dozens of these "useless" spaceport encounters may discourage newer players from checking the spaceport whenever possible, and so they may miss out on missions or whole storylines, because one of these has "taken up" a spaceport where otherwise a mission chain would be offered. Don't see how that could be prevented given there's no "minor+" tag to have something be of lower priority than chains that may already have the "minor" tag

I haven't checked all of Lake/ @Galaucus 's stuff just yet, but they're only meant to have like a 1%(max 5%) chance of firing. So the odds of them taking up another spot should be very small.

Arachi-Lover commented:
But if there'll be ~10 of these per faction then that's quite higher than 1% in total. Given Katya's first mission is pretty low
`	to offer`
`		random < 10 + 7 * "assisted free worlds" * "assisted free worlds"`
then these will significantly interfere with chances of getting a story mission or not.

MasterOfGrey commented:
> But if there'll be ~10 of these per faction then that's quite higher than 1% in total. Given Katya's first mission offer% is pretty low
> ` to offer`
> ` random < 10 + 7 * "assisted free worlds" * "assisted free worlds"`
> then these will significantly interfere with chances of getting a story mission or not.

That's not strictly how the rng works... If we have 10 of them there's a 10% chance on any given landing that we get 1 of them, but each successive landing reduces the chance of getting another one. Getting Katya without doing any assistance is still as likely as getting just 1 out of 10 of these events.
Further, each roll is individual and, to my understanding, checked alphabetically. So things like Katya whose mission starts with "FW" will roll before anything else starting further down the alphabet. She's comparatively more likely to exclude one of these than the reverse.

But if you're very concerned we could just put a Z at the front of every mission name, then they'd almost never block anything.

Anarchist2 commented:
Kinda iffy on having independent culture, since there's only 2 systems iirc that become independent during FW.

MasterOfGrey commented:
There maybe rather fewer of those.

MasterOfGrey commented:
My current intentions are to have the independent ones all specify ***a*** planet and provide flavour only in the event that they become independent.

trerri commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+	on offer
+		conversation
+			`Walking through the spaceport you come across the uncommon pairing of a Hai and a human, their conversation standing out by virtue of speaking the common tongue of humanity.`
+			`	"Oh, oh, Gemanari this looks interesting!" The human woman looks to be in her late sixties and, when you turn to look, seems to be clucking over a holographic advertisement for a theater show. "This looks adorable doesn't it? Oh and it says they're using ripple ray refractive holographic flight effects!"`
```
Missing comma after "Gemanari" and "adorable" and "Oh"

trerri commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+			`	"Oh, oh, Gemanari this looks interesting!" The human woman looks to be in her late sixties and, when you turn to look, seems to be clucking over a holographic advertisement for a theater show. "This looks adorable doesn't it? Oh and it says they're using ripple ray refractive holographic flight effects!"`
+			`	The advertisement does indeed look very impressive and you pause to observe for a moment.`
+			`	The Hai, presumably Gemanari, also appears to be a more mature lady who squints at it and wrinkles her nose. "Ugh, that's the Germontian interpretation though."`
+			`	This receives a puzzled look to which Gemanari thinks for a second before clarifying, "You would say the Darwinist interpretation."`
```
Make her say social Darwinist

trerri commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+					goto wrapup
+				`	"Whats the Musical Entertainment Expo?"`
+				`	"Why don't they start drinking yet?"`
+			`	"Well you see the big music labels do a lot of talent scouting all year round, they all have a department that is solely dedicated to finding and assembling new music groups to be the next big thing. It's traditionally to celebrate only after you've committed to the journey with a new band. Every three years the industry holds ME2, that's what you're seeing up there, to debut their next roun ...
```
Missing comma after "you see".
Missing conjunction before "they all have".
Why would a Hai event have an acronym that makes sense in English?
Replace "that's what you're seeing" with "which is what you're seeing".

MasterOfGrey replied:
I disagree with the missing conjunction, it's speech after all and it's not necessary to include it when spoken.
It's heavily implied that the Hai have a pretty decent education system and learn Human standard - I don't see it as much of a reach that they could re-acronym a name after they've translated the name across.

trerri commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+		random == 1
+	on offer
+		conversation
+			`Stepping into the spaceport you are beset by a riot of color. Every blank surface that's not usually paid advertising space seems to be festooned with dangling holographic projectors, ribbons, banners, and even actual paper fliers and posters, apparently taped to whatever surface will bear them. At first you wonder if there's some sort of festival going on, but as you watch a young Hai wearin ...
```
Use "anthropomorphic". If you use "anthro" the first thing the player's going to think is "Is the dev a furry?".
The banner-tearing and goo-slapping are a bit too fast-paced and out of the blue to look elegantly written.

MasterOfGrey replied:
Changed anthro, and it's meant to be fast-paced and inelegant. The idea is that it's happening almost too quickly for you to follow exactly what's going on because you've just walked into the equivalent of a warzone at an anime gaming convention.

trerri commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+			`Stepping into the spaceport you are beset by a riot of color. Every blank surface that's not usually paid advertising space seems to be festooned with dangling holographic projectors, ribbons, banners, and even actual paper fliers and posters, apparently taped to whatever surface will bear them. At first you wonder if there's some sort of festival going on, but as you watch a young Hai wearin ...
+			branch translator
+				has "language: Wanderer"
+			`	As you watch the triangle elder being holoprojected seems to flicker and then fizzles out as the battery unit fades, apparently under the assault of the root vegetable which appears to do a victory dance, before proceeding to act out something that makes precisely no sense to you.`
```
Rewrite this from the ground up, it makes no sense.

MasterOfGrey replied:
I assume you're referring to line 86, it's an e-warfare device inside a holoprojector. When the projected memes do battle the underlying computers are also trying to kill each other. One wins.

trerri replied:
I mean that it's written and conveyed poorly.

MasterOfGrey replied:
It has edited based on vkr's feedback, so maybe look there, but beyond that I'm afraid unless you can be more specific I can't see particularly what the complaint is.

trerri commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+				has "language: Wanderer"
+			`	As you watch the triangle elder being holoprojected seems to flicker and then fizzles out as the battery unit fades, apparently under the assault of the root vegetable which appears to do a victory dance, before proceeding to act out something that makes precisely no sense to you.`
+			`	A little further on you encounter two different groups of protesters weaving in and out of the spaceport's entrances like some kind of conga line. The triangular elder and root vegetable make several reappearances on their placards, some of them holographically animated.`
+			`	A different security guard, this one in regular attire, appears at your elbow. "You look a little lost sir but don't worry, it's just a meme war. Carry on with your business and don't touch the meme items, no one should bother you over it."`
```
I hate to say "cringe" in 2021, but **this is cringe.**
Aside from the usual wonky pacing, DO NOT USE THE WORD "MEME" IN THE GAME PLEASE I BEG YOU.

*cough* sorry i lost my cool

MasterOfGrey replied:
Meme might be popularised right now, but it's a legitimate term to describe the phenomenon. In fact there is no equivalent term in english which can be substituted and this is a rare case where society has popularised the correct scientific use of a term.

trerri commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+			`	You thank the security guard for his advice and go about what you were doing as the baffling conflict continues around you.`
+				decline
+			label translator
+			`	As you watch the triangle elder being holoprojected seems to flicker and then fizzles out as the battery unit fades, apparently under the assault of the root vegetable which appears to do a victory dance. To your surprise your translator pipes up, "[Victory, Success] for the true and square root that is more [significant, meaningful] than the tripartite [system, tradition] of the elders!"`
```
Translator output is weird, but it could be excused by it not being very good at picking up Hai slang.
Do the Hai even have slang?

MasterOfGrey replied:
This is how the translator translates Hai, you only otherwise use the translator when the Wanderer's speak Hai to you, because the Hai you encounter generally have learned human standard.

trerri replied:
I mean that "for the true and square root that is more [significant, meaningful]" is an awkward construction.

MasterOfGrey replied:
Ah yes, well in this case consider your translator is essentially transliterating meme text.

trerri commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+		random == 1
+	on offer
+		conversation
+			`Upon entering a diner at the spaceport you have not been seated long before you notice there is a group of other human ship captains sequestered in a corner playing, near as you can tell, a game of poker. You note that the group seems to be getting a variety of dark looks from various Hai also dining at the establishment.`
```
"Upon entering" and "you have not been seated long" directly conflict.
I was not aware "near as you can tell" was a thing, use "as far" instead possibly.

MasterOfGrey replied:
Re-done the first part.
R.E. "as far" -> I have a little bit of a personal crusade against simplifying language for the sake of simplifying language in a _text based_ game. So if the phrasing is legitimate and the meaning is clear, I'm in favour of keeping it.

trerri commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+			`Upon entering a diner at the spaceport you have not been seated long before you notice there is a group of other human ship captains sequestered in a corner playing, near as you can tell, a game of poker. You note that the group seems to be getting a variety of dark looks from various Hai also dining at the establishment.`
+			`	After a short while the barkeep, a middling and slightly overweight Hai, comes over to take your order.`
+			choice
+				`	"Just some food thanks." (Place an order.)`
```
Aside from the missing comma, that's a weird order. I suggest just having (Place an order.)

trerri commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+			choice
+				`	"Just some food thanks." (Place an order.)`
+					goto end
+				`	"Hey, I notice there are some dark looks around here. Are human captains not welcome?"`
```
Repetition of "dark looks", wrong tense on "I notice".

MasterOfGrey replied:
If you noticed what you'd think of as 'dark looks' would you not then use the words 'dark looks' when asking about it?
Fixed the tense.

trerri replied:
I don't care if you would realistically use the word, repetitions are bad.
The player will see the repetition and see it as ugly.

trerri commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+				`	"Hey, I notice there are some dark looks around here. Are human captains not welcome?"`
+			`	The barkeep glances over at the table and shakes his head, "Oh no, you're perfectly welcome. That group is just breaking the maturity tenets, some Hai get uncomfortable with adults blatantly acting like children in public."`
+			choice
+				`	"OK..." (Place an order.)`
```
No idea why but "OK" seems out of place for a conversation option in ES. Just a gut feeling.

MasterOfGrey replied:
Removed for consistency with previous choice.

trerri commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+				`	"OK..." (Place an order.)`
+					goto end
+				`	"The maturity tenets?"`
+			`	"Oh right, I suppose you wouldn't know, there are a number of tenets that Hai are taught growing up, lessons to be internalized to be a responsible and functioning member of society. The second is 'To be honest in dealings.' and the third is 'To gamble only on skill, not chance.' so their game breaks two of them."`
```
Replace comma with period after "I suppose you wouldn't know".
"to be internalized to be" has a repetition but it isn't particularly bad.
Put commas after quotes: "The second is 'To be honest in dealings.', and the third is 'To gamble only on skill, not chance.', so their game breaks two of them."

MasterOfGrey replied:
I knew I'd bugger the quotes somehow.

trerri commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+			`	"Oh right, I suppose you wouldn't know, there are a number of tenets that Hai are taught growing up, lessons to be internalized to be a responsible and functioning member of society. The second is 'To be honest in dealings.' and the third is 'To gamble only on skill, not chance.' so their game breaks two of them."`
+			choice
+				`	"Some would say Poker is more about skill than chance."`
+				`	"Surely if it's just a game though, there's no harm in it?"`
```
It's fine but maybe could sound better if it was more like:
"Surely though, if it's just a game, there's no harm in it?"

MasterOfGrey replied:
I personally, when speaking aloud, would only use the two-part sentence.
I agree though, that if it was written as if it were writing, the three-part structure makes sense.

trerri commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+				`	"Wait, do the Hai not lie or bluff?"`
+			`	The barkeep chortles with a strange trill and says, "Oh no the Hai do lie and bluff, we're actually very good at bluffing, evolutionarily advantageous in the wild and all that. It's just seen as very immature and base. An adult should be able to conduct their lives without resorting to such primitive instincts. We learned that the hard way long ago, back before we holed up in our current bor ...
+			`	He looks at you intently before continuing, "You won't hear the tenets brought up very much, but if you look closely you'll see that the first three especially are ingrained in our society very deeply."`
+			`	With a shrug he flips open his order pad and pats his belly. "Of course, my gran always said that any of them after the tenth were more guidelines than actual rules, and I certainly have my share of disagreements with thirteen. Now, before I talk your ears of, what'll I get for you?"`
```
Did you mean "the thirteen*th*"?
Misspelling of "off".

MasterOfGrey replied:
I did not, it is common linguistic shorthand to refer to familiar entities on a list of distinct and individually separate items by 'name' rather than by position. 'Thirteen' is the 'name' of the thirteenth tenet.
Fixed spelling.

trerri replied:
Use "the thirteen" or "number thirteen". It's just ugly how it is right now.

trerri commented on data/human/culture free worlds.txt:
```diff
@@ -0,0 +1,67 @@
+			`	"Back in the forties," a reveler drunkenly explains to you, "Half the farms'n'ranches'honesfolks about this planet were either about ready to leave or takin' out a second mortgage on their property. Bankers from richer worlds were always comin' about to repossess something or another." A passer-by spits on the ground in disgust at the mention - or perhaps he'd just succeeded in extricating s ...
+			`	By this point several others have gathered around the speaker. It would appear that recounting the tale of the Great Pig War to strangers is part of the tradition. "... Mnh. S'anyways. Some boys flew over to Maker with a hundred-"`
+			`	The speaker is interrupted by a shout of "It was a thousand!"`
+			`	"Quiet down! Ain't no way you could fit a thousand swine onto a hauler. Alright, so they've got two hundred pigs in their hold when they try to land on Maker. Syndicate Security shows up-" Boos echo throughout the crowd "- and tell these good, upright, honest smugglers that they're to match velocity n' prepare to be boarded. Now, they aren't about to bow down, so they make a break for the ed ...
```
hole*

trerri commented on data/human/culture free worlds.txt:
```diff
@@ -0,0 +1,67 @@
+			`	The speaker is interrupted by a shout of "It was a thousand!"`
+			`	"Quiet down! Ain't no way you could fit a thousand swine onto a hauler. Alright, so they've got two hundred pigs in their hold when they try to land on Maker. Syndicate Security shows up-" Boos echo throughout the crowd "- and tell these good, upright, honest smugglers that they're to match velocity n' prepare to be boarded. Now, they aren't about to bow down, so they make a break for the ed ...
+			`	The listeners go quiet. Tense anticipation fills the air. If not for the jolly music and overwhelming clamor of the crowd surrounding your little huddle, all would be hushed. "And that's when nine green-striped angels punched outta' hyperspace and began streaking towards our little drama. See, some of the local militia decided that justice was more important than jurisdiction, and they decid ...
+				decline
```
Maybe have the crowd clamor a little at the end of the narration.

trerri commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+			`	He looks at you intently before continuing, "You won't hear the tenets brought up very much, but if you look closely you'll see that the first three especially are ingrained in our society very deeply."`
+			`	With a shrug he flips open his order pad and pats his belly. "Of course, my gran always said that any of them after the tenth were more guidelines than actual rules, and I certainly have my share of disagreements with thirteen. Now, before I talk your ears of, what'll I get for you?"`
+			label end
+			`	He takes your order diligently and with a smile.`
```
I don't seem to recall that the ending label ever works like this, it's usually just (done) or something in the first person.

MasterOfGrey replied:
The label endings are just code hooks in the background. They're just matching strings for the goto-label combo.

trerri replied:
No no, I was referring to "He takes your order diligently and with a smile."
No other conversation ends in the second person like that.

MasterOfGrey replied:
Oh, hmmm.

MasterOfGrey replied:
Ah right, so you can't go directly from a label to a `decline` to end. So in order for you to skip the explanation the mission _needs_ some text there.
If you can think of an alternative way of doing it please submit a suggestion. :)

trerri replied:
Just have the final one also say (Place an order.) and when you choose it it just says 'The bartender takes your order with a smile.'

MasterOfGrey replied:
I don't see how what you're proposing is meant to work. Why don't you _insert a suggestion_ which demonstrates it?

trerri commented on data/human/culture republic.txt:
```diff
@@ -0,0 +1,125 @@
+		random == 1
+	on offer
+		conversation
+			`Passing through the spaceport you see a sizable group of people waiting on a lottery draw. You hear a whoop of success and your attention is drawn as one of them wins a moderate sum in an 12th division win (a day or two worth of standard pay).`
```
a* 12th division win
Replace (a day or two worth of standard pay) with "which corresponds to a day or two of standard pay".

vkr-13 replied:
Quotations would imply that someone spoke them? I think it would be more fitting to say `a moderate sum in an 12th division win; a day or two worth of standard pay.`

MasterOfGrey replied:
That's what I get for changing it from an 'eighth' division win without proofreading.
Thanks vkr, went with that.

trerri commented on data/human/culture republic.txt:
```diff
@@ -0,0 +1,125 @@
+		random == 1
+	on offer
+		conversation
+			`Visiting the Paradise spaceports is always an interesting experience for someone from abroad. Nowhere else is class such an ingrained component of daily life. You catch yourself noticing people who seem to be invisible to most of those going about their business, but they never catch your eye. Perhaps it is because of your background, but you notice them even when they are avoiding notice, an ...
```
Repetition of "notice".
Also, are the "invisible" people easy to notice for first last or hard? It's really not clear.

MasterOfGrey replied:
Removed one notice, but this circumstance is a little bit like that. Hopefully made it more clear.

trerri commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# "Missions" which add culture events to the spaceport.
+
```
Disclaimer: I absolutely love most of this, but I'm still going to be ruthless in the criticism so it can be made even better.

trerri reviewed (commented):
Reviewed everything <3

vkr-13 commented on data/human/culture republic.txt:
```diff
@@ -0,0 +1,125 @@
+		conversation
+			`As you wander the spaceport with a snack, waiting for your ship to be refueled, you pass a local souvenir shop and poke your head in to kill a little time. You see a decent looking mug with the text "<source>" printed underneath an attractive, stylized representation of the spaceport you are presently standing in and pick it up to see how it feels. The handle is solid, and not too small for y ...
+			`	Two young adults wander into the aisle just a couple of meters away. They're clearly from abroad judging by their conversation occurring in some local dialect which is neither local nor one you understand. They're cooing over an item they've found but when one flips it over to look at the price tag there is an immediate exchange of looks and expressions that does not require translation. The ...
+			`	Curious you turn over your own mug and see that it is indeed at least five times what you would expect to pay for a mug, even a good one. Still, t is certainly a novelty...`
```
`	Curious you turn` should be `	Curious, you turn`
`Still, t is certainly a novelty...` should be `Still, it is certainly a novelty...`

vkr-13 commented on data/human/culture syndicate.txt:
```diff
@@ -0,0 +1,57 @@
+		random == 1
+	on offer
+		conversation
+			`You take a shortcut down an alley and are shocked to find several people with wheelchairs or crutches lving in a shanty town along a hidden service road. It seems Syndicate worlds don't have much in the way of welfare systems. (TBD)`
```
I'm assuming TBD means this requires further work?

MasterOfGrey replied:
Yes - it's just my cliffnotes version. Feel free to put forward a more complete text suggestion, or it'll likely get removed and revisited in a future PR.

vkr-13 reviewed (changes requested):
There's a lot to like, but there's also room to improve. Specifically a large majority in the Hai missions felt either rushed or like they had been edited several times mid-creation.

Looking forward to the end results!

MasterOfGrey commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+			`	"Oh, oh, Gemanari this looks interesting!" The human woman looks to be in her late sixties and, when you turn to look, seems to be clucking over a holographic advertisement for a theater show. "This looks adorable doesn't it? Oh and it says they're using ripple ray refractive holographic flight effects!"`
+			`	The advertisement does indeed look very impressive and you pause to observe for a moment.`
+			`	The Hai, presumably Gemanari, also appears to be a more mature lady who squints at it and wrinkles her nose. "Ugh, that's the Germontian interpretation though."`
+			`	This receives a puzzled look to which Gemanari thinks for a second before clarifying, "You would say the Darwinist interpretation."`
```
Alternate change made.

MasterOfGrey commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+					goto wrapup
+				`	"Whats the Musical Entertainment Expo?"`
+				`	"Why don't they start drinking yet?"`
+			`	"Well you see the big music labels do a lot of talent scouting all year round, they all have a department that is solely dedicated to finding and assembling new music groups to be the next big thing. It's traditionally to celebrate only after you've committed to the journey with a new band. Every three years the industry holds ME2, that's what you're seeing up there, to debut their next roun ...
```
Cheers, and the choice of ME2 is a subtle reference to E3.

MasterOfGrey commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+		random == 1
+	on offer
+		conversation
+			`While stopped in the Spaceport waiting for refueling you are surprised by how hard it is to find somewhere to sit and have a meal & a drink. It seems that Hai are crammed into every bar or similar venue with a public display screen, arrayed in sashes and mantles and all manner of other paraphernalia which, near as you can tell looks like... team sport merchandise?`
```
I'm going to keep 'stopped', have accepted the rest.

MasterOfGrey commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+	on offer
+		conversation
+			`While stopped in the Spaceport waiting for refueling you are surprised by how hard it is to find somewhere to sit and have a meal & a drink. It seems that Hai are crammed into every bar or similar venue with a public display screen, arrayed in sashes and mantles and all manner of other paraphernalia which, near as you can tell looks like... team sport merchandise?`
+			`	You're not really an expert on exactly what such things look like in Hai society and so when you finally find somewhere to sit, at a bar for lack of any other options, you ask the tired looking bartender what exactly is going on.`
```
The one time I go for simpler language and someone suggests I use more advanced vocabulary. :P 

While I would use -'s personally in messages, I think that's frowned upon in the ES style directions.

MasterOfGrey commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+		random == 1
+	on offer
+		conversation
+			`Upon entering a diner at the spaceport you have not been seated long before you notice there is a group of other human ship captains sequestered in a corner playing, near as you can tell, a game of poker. You note that the group seems to be getting a variety of dark looks from various Hai also dining at the establishment.`
```
I think the original phrasing emphasises that the group of human captains are a minority in the room. I'm not against this change though if people want it.

MasterOfGrey commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+				`	"Do you not have musicians trying to make it on their own?"` 
+			`	"Oh for sure, but mostly just to get picked up by one of these companies."`
+			label production
+			`	The bartender chuckles. "We've been doing pop music for longer than your people have had fire, it's about the experience of being a fan of a group as it rises to fame and stardom. A dedicated Hai can go back and find a song for almost any possible life experience in almost any possible genre. It's the journey, and the personalities, that are what it's all about."`
```
In this particular case I'm going to disagree, but only because of the intended emphasis.
The entire paragraph is in the context of 'the experience'. 
`being a fan as a group rises` shares the emphasis with the group, the original links the event of `it rising` to `the experience`.
Similarly the language choice in the last sentence is meant to emphasise that it is specifically these things that are are the purpose, and detract from other factors.
I could change 'that are what it's all about' though, I sorta get that that might be excessive.

MasterOfGrey commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+			`Stepping into the spaceport you are beset by a riot of color. Every blank surface that's not usually paid advertising space seems to be festooned with dangling holographic projectors, ribbons, banners, and even actual paper fliers and posters, apparently taped to whatever surface will bear them. At first you wonder if there's some sort of festival going on, but as you watch a young Hai wearin ...
+			branch translator
+				has "language: Wanderer"
+			`	As you watch the triangle elder being holoprojected seems to flicker and then fizzles out as the battery unit fades, apparently under the assault of the root vegetable which appears to do a victory dance, before proceeding to act out something that makes precisely no sense to you.`
```
It's not coincidental, there's e-warfare happening in the background.
`triangular` is definitely a good catch, will make a pass on that.

MasterOfGrey commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+		random == 1
+	on offer
+		conversation
+			`Stepping into the spaceport you are beset by a riot of color. Every blank surface that's not usually paid advertising space seems to be festooned with dangling holographic projectors, ribbons, banners, and even actual paper fliers and posters, apparently taped to whatever surface will bear them. At first you wonder if there's some sort of festival going on, but as you watch a young Hai wearin ...
```
I have altered the first paragraph for clarity incorporating some of your suggestion.
The last bit I'm hesitant to change because 'continues to vandalize' sounds like the Hai has time to do several things, when I want to give the impression that they had only literal seconds before security started chasing them.

MasterOfGrey commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+			choice
+				`	"Just some food thanks." (Place an order.)`
+					goto end
+				`	"Hey, I notice there are some dark looks around here. Are human captains not welcome?"`
+			`	The barkeep glances over at the table and shakes his head, "Oh no, you're perfectly welcome. That group is just breaking the maturity tenets, some Hai get uncomfortable with adults blatantly acting like children in public."`
```
I have adopted the second part of this suggestion, I think the experience of the Hai is sufficient to say that <first> would have to be a dumbass to assume it might be 'all humans'.

MasterOfGrey commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+				`	"OK..." (Place an order.)`
+					goto end
+				`	"The maturity tenets?"`
+			`	"Oh right, I suppose you wouldn't know, there are a number of tenets that Hai are taught growing up, lessons to be internalized to be a responsible and functioning member of society. The second is 'To be honest in dealings.' and the third is 'To gamble only on skill, not chance.' so their game breaks two of them."`
+			choice
+				`	"Some would say Poker is more about skill than chance."`
+				`	"Surely if it's just a game though, there's no harm in it?"`
+			`	"Oh no, I'm afraid the first tenet ensures that it's really not favorable either way. The first is 'To have clear separation of fantasy and reality.' so a game cannot have real consequences or it is reality, not fantasy. So to gamble money on the game is to bring real consequences to a fantasy exercise, this is taboo for an adult.`
```
I'm just going to say - People don't necessarily speak and present information logically.
But if you think there are a collection of grammar edits needed please use the function to suggest en edit.

trerri replied:
> I'm just going to say - People don't necessarily speak and present information logically.

This is a terrible approach to writing anything. In movies people don't interrupt others, forget what they're saying, etc. unless it serves the plot. If someone coughs in a story they are either dying or they have some disease which is important to the plot. This is not a flaw, it is by design. You shouldn't excuse wonky writing with "realism", because if you actually wanted to write exactly how people speak it would suck to read.

MasterOfGrey commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,208 @@
+				`	"Wait, do the Hai not lie or bluff?"`
+			`	The barkeep chortles with a strange trill and says, "Oh no the Hai do lie and bluff, we're actually very good at bluffing, evolutionarily advantageous in the wild and all that. It's just seen as very immature and base. An adult should be able to conduct their lives without resorting to such primitive instincts. We learned that the hard way long ago, back before we holed up in our current bor ...
+			`	He looks at you intently before continuing, "You won't hear the tenets brought up very much, but if you look closely you'll see that the first three especially are ingrained in our society very deeply."`
+			`	With a shrug he flips open his order pad and pats his belly. "Of course, my gran always said that any of them after the tenth were more guidelines than actual rules, and I certainly have my share of disagreements with thirteen. Now, before I talk your ears of, what'll I get for you?"`
```
See response to trerri re this.

MasterOfGrey commented on data/human/culture free worlds.txt:
```diff
@@ -0,0 +1,67 @@
+	on offer
+		conversation
+			`You arrive at the spaceport just in time to witness a shuttle being towed off its landing pad by a tractor, a familiar blazon displayed upon the spacecraft's doors. Though it's made somewhat harder to recognize by the fact that somebody has spray-painted "REPOSESED" over the logo, it appears that this shuttle belongs to - or used to belong to - the very same galactic bank that cut you the loa ...
+				decline
```
@Galaucus (this isn't mine)

MasterOfGrey commented on data/human/culture free worlds.txt:
```diff
@@ -0,0 +1,67 @@
+	on offer
+		conversation
+			`As you exit your ship, you run overhear a spaceport technician attempting to have an animated conversation with another. "Piracy? And the militia does nothing? That goes against everything we stand for." Stern disapproval colors the speaker's voice. "These clowns're assaulting bankers and taking their shuttles, last I heard. I get that they're frustrated out there on the farms and owe money,  ...
+				decline
```
@Galaucus (this isn't mine)

MasterOfGrey commented on data/human/culture pirate.txt:
```diff
@@ -0,0 +1,11 @@
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# "Missions" which add culture events to the spaceport.
```
Yeah this file will probably be removed and re-added in a future PR.

Anarchist2 commented:
What's with the double Z at the start of each mission definition?

ESCelestia reviewed (changes requested):
Suggested:

mission "ZZCU Tarazed Ship Christening"
	minor
	source "Wayfarer"
	to offer
		random == 1
		has "main plot completed"
	on offer
		conversation
			`As you walk past a number of ships and crew going about their work in the spaceport, a small gathering in front of a Shuttle catches your attention...`

MasterOfGrey commented:
> What's with the double Z at the start of each mission definition?

It ensures that other missions that actually have consequence are checked for availability before these are. (Even other minor missions.)

Zitchas commented:
Clever solution.

I was thinking of asking, along the same lines, if a mission that triggers on landing would block a spaceport mission, or can we have one fire in each location and thus not block each other?

MasterOfGrey commented:
> Clever solution.
> 
> I was thinking of asking, along the same lines, if a mission that triggers on landing would block a spaceport mission, or can we have one fire in each location and thus not block each other?

None of them are meant to be on landing, I'm waiting on @Galaucus /Lake to get back to his content which uses that.

Amazinite commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+#          No Translator Required          #
+############################################
+
+mission "ZZCU Theater Chat"
```
It seems to me like the strict offer conditions of most of these missions fulfil the purpose of making sure they don't get offered over other missions. Even ignoring that though, I don't see an issue with sometimes getting these missions over other missions. It's not like these ones repeat. Therefore, I wouldn't have these extra characters in the mission names just to push the mission to the end of the offer list. 

I'd remove the comments, like the one above. They seem unnecessary to me. There's no need to comment to organize these missions, or to say who worked on them, since we know that from the PR and ultimately the changelog attribution. Whatever the case on comments, we should have ones like the "No Translator Required" comment above, as that actually goes against the coding style guide, so I'd avoid it in the data files as well.

> Fancy, decorated comments should not be used.
> For example, perhaps your editor has a "feature" that lets you automatically insert comments like this:
```
/*************************************
 * My fancy comment.
 * This is a comment that could have
 * been a single line without these
 * fancy decorative elements.
 *************************************/
```
> But, those two lines of asterisks are eye-grabbing and completely unnecessary on any editor that supports syntax coloring, and more importantly anyone using a different editor would have to manually type an asterisk at the beginning of every comment line and figure out exactly how many asterisks are in those rows at the top and bottom, in order to be consistent with your code.

Having separate files just for these missions also seem unnecessary.  These could simply be put into the existing mission files. "culture hai.txt" gets added to "hai mission.txt," the rest just get added to "transport mission.txt" (even though they aren't actually transport missions, but that file has already had missions added that aren't transport related).

Zitchas replied:
"transport missions.txt" = "general purpose mission file for non specific missions"

MasterOfGrey replied:
The ZZ was mostly because the Spider had raised some strident objections to them being in a position to block other missions, since these are all missions that are specifically of no consequence.

This was also why they're in their own files. There's going to be more of them, and all the other missions in the game so far have some sort of consequence. Whether it's a reward, or a change in the world, or they unlock other missions or dialogue options. All the other missions _do_ something, and these only provide world-building insight to the player. There's only 23 missions across the set so far, but Lake is doing some more and the plan is to come back with a second batch and so on until (across this scope) there's about 180 of them. 

I really think it's better if we don't put a clutter of no-consequence fluff missions in the same files with the ones that actually do things. If only for the benefit of content creators and their sanity when trying to sort through things.

Amazinite replied:
If we want these is separate files, then at the very least combine all the human ones. The distinction between regions/sources into different files is overkill to me. And how about instead of calling them “culture missions,” we call them “spaceport conversations.txt,” with the Hai one being “hai spaceport conversations.txt”?

Amazinite reviewed (commented):
Just commenting on the setup right now.

Amazinite commented on data/human/culture free worlds.txt:
```diff
@@ -0,0 +1,67 @@
+			`Farm world spaceports tend to be lonely affairs outside of harvesting season, but this one is bustling with festive energy. Rows of colorful tents and throngs of the planet's natives fill the area, as well as a proliferation of craft-work pig-sculptures; some are built from hay, some papier-mache and paint - one even appears to have been carved from butter. Most spaces that aren't taken by a  ...
+			choice
+				`	(Carry on with your business. You can always come back next year.)`
+					defer
```
I'd make this a decline thing, but that goes against the choice of "I can come back later," so maybe not.
```suggestion
					decline
```

MasterOfGrey replied:
This is the start of a longer chain of conversations, so I think it would be a shame to make this a decline.

Amazinite commented on data/human/culture free worlds.txt:
```diff
@@ -0,0 +1,67 @@
+
+mission "ZZCU South: The Great Pig War"
+	landing
+	to offer
+		and
+			month == 6
+			day >= 10
+			day <= 20
+	source
+			attributes "spaceport" "farming" "south"
```
```suggestion
	source
			attributes "farming" "south"
	to offer
		and
			month == 6
			day >= 10
			day <= 20
```

Amazinite commented on data/human/culture free worlds.txt:
```diff
@@ -0,0 +1,67 @@
+# "Missions" which add culture events to the spaceport.
+
+mission "ZZCU South: The Great Pig War"
+	landing
```
```suggestion
	minor
```

Amazinite commented on data/human/culture free worlds.txt:
```diff
@@ -0,0 +1,67 @@
+		conversation
+			`Farm world spaceports tend to be lonely affairs outside of harvesting season, but this one is bustling with festive energy. Rows of colorful tents and throngs of the planet's natives fill the area, as well as a proliferation of craft-work pig-sculptures; some are built from hay, some papier-mache and paint - one even appears to have been carved from butter. Most spaces that aren't taken by a  ...
+			choice
+				`	(Carry on with your business. You can always come back next year.)`
```
"Action" choices (those in parentheses) are always written in first person perspective.
Important to note though that we don't actually know that this is a yearly festival.
```suggestion
				`	(Carry on with your business. I can always come back next year.)`
```

Amazinite commented on data/human/culture free worlds.txt:
```diff
@@ -0,0 +1,67 @@
+				`	(Carry on with your business. You can always come back next year.)`
+					defer
+				`	(Stop and take in the sights - and maybe a soft pretzel or three.)`
+			`	Conversation with the locals eventually turns to the matter of what they're celebrating in the first place. As it turns out, the bronzed pig holds the clue. Today is the anniversary of the Great Pig War, a conflict which apparently raged between the Southern Defense Pact and the Syndicate unbeknownst to the larger galaxy. Toasts are raised in honor of nine heroic captains - and to the memory ...
```
The southern defense pact is a more recent thing, and is also strictly about defending the south form pirates. Maybe change the two parties to just the southern worlds vs the Syndicate.

Amazinite commented on data/human/culture free worlds.txt:
```diff
@@ -0,0 +1,67 @@
+					defer
+				`	(Stop and take in the sights - and maybe a soft pretzel or three.)`
+			`	Conversation with the locals eventually turns to the matter of what they're celebrating in the first place. As it turns out, the bronzed pig holds the clue. Today is the anniversary of the Great Pig War, a conflict which apparently raged between the Southern Defense Pact and the Syndicate unbeknownst to the larger galaxy. Toasts are raised in honor of nine heroic captains - and to the memory ...
+			`	"Back in the forties," a reveler drunkenly explains to you, "Half the farms'n'ranches'honesfolks about this planet were either about ready to leave or takin' out a second mortgage on their property. Bankers from richer worlds were always comin' about to repossess something or another." A passer-by spits on the ground in disgust at the mention - or perhaps he'd just succeeded in extricating s ...
```
```suggestion
			`	"Back in the forties," a reveler drunkenly explains to you, "half the farms 'n' ranches hones' folks about this planet were either about ready to leave or takin' out a second mortgage on their property. Bankers from richer worlds were always comin' about to repossess something or another." A passer-by spits on the ground in disgust at the mention - or perhaps he'd just succeeded in extricating some shredded pork from his teeth. "So some good honest boys decided they'd had enough of payin' tariffs and honoring exclusive shipping rights n' rot like that. They came 'round, bought up a whole herd of swine, and took off to feed some hungry folks in the Syndicate. Trouble is, back in those days the Syndicate had a stranglehold on shipping grain n' livestock. Either you signed up with this one shipping company and let them eat all your profits, or you let your harvest rot. Getting to market without being a member was... somewhat frowned upon."`
```

Amazinite commented on data/human/culture free worlds.txt:
```diff
@@ -0,0 +1,67 @@
+			`	Conversation with the locals eventually turns to the matter of what they're celebrating in the first place. As it turns out, the bronzed pig holds the clue. Today is the anniversary of the Great Pig War, a conflict which apparently raged between the Southern Defense Pact and the Syndicate unbeknownst to the larger galaxy. Toasts are raised in honor of nine heroic captains - and to the memory ...
+			`	"Back in the forties," a reveler drunkenly explains to you, "Half the farms'n'ranches'honesfolks about this planet were either about ready to leave or takin' out a second mortgage on their property. Bankers from richer worlds were always comin' about to repossess something or another." A passer-by spits on the ground in disgust at the mention - or perhaps he'd just succeeded in extricating s ...
+			`	By this point several others have gathered around the speaker. It would appear that recounting the tale of the Great Pig War to strangers is part of the tradition. "... Mnh. S'anyways. Some boys flew over to Maker with a hundred-"`
+			`	The speaker is interrupted by a shout of "It was a thousand!"`
```
```suggestion
			`	The speaker is interrupted by a shout of, "It was a thousand!"`
```

Amazinite commented on data/human/culture free worlds.txt:
```diff
@@ -0,0 +1,67 @@
+			`	"Back in the forties," a reveler drunkenly explains to you, "Half the farms'n'ranches'honesfolks about this planet were either about ready to leave or takin' out a second mortgage on their property. Bankers from richer worlds were always comin' about to repossess something or another." A passer-by spits on the ground in disgust at the mention - or perhaps he'd just succeeded in extricating s ...
+			`	By this point several others have gathered around the speaker. It would appear that recounting the tale of the Great Pig War to strangers is part of the tradition. "... Mnh. S'anyways. Some boys flew over to Maker with a hundred-"`
+			`	The speaker is interrupted by a shout of "It was a thousand!"`
+			`	"Quiet down! Ain't no way you could fit a thousand swine onto a hauler. Alright, so they've got two hundred pigs in their hold when they try to land on Maker. Syndicate Security shows up-" Boos echo throughout the crowd "- and tell these good, upright, honest smugglers that they're to match velocity n' prepare to be boarded. Now, they aren't about to bow down, so they make a break for the ed ...
```
```suggestion
			`	"Quiet down! Ain't no way you could fit a thousand swine onto a hauler. Alright, so they've got two hundred pigs in their hold when they try to land on Maker. Syndicated Security shows up-" Boos echo throughout the crowd "- and tell these good, upright, honest smugglers that they're to match velocity n' prepare to be boarded. Now, they aren't about to bow down, so they make a break for the edges of the system but start taking hits. Quicksilvers are buzzin', Protectors lumberin' after them, and their shields go down. A particle cannon punches straight through the hull, and our good, kind, noble, brave boys n' girls stop their ship and start patchin' the hole, ready to surrender."`
```

Amazinite commented on data/human/culture free worlds.txt:
```diff
@@ -0,0 +1,67 @@
+			`	By this point several others have gathered around the speaker. It would appear that recounting the tale of the Great Pig War to strangers is part of the tradition. "... Mnh. S'anyways. Some boys flew over to Maker with a hundred-"`
+			`	The speaker is interrupted by a shout of "It was a thousand!"`
+			`	"Quiet down! Ain't no way you could fit a thousand swine onto a hauler. Alright, so they've got two hundred pigs in their hold when they try to land on Maker. Syndicate Security shows up-" Boos echo throughout the crowd "- and tell these good, upright, honest smugglers that they're to match velocity n' prepare to be boarded. Now, they aren't about to bow down, so they make a break for the ed ...
+			`	The listeners go quiet. Tense anticipation fills the air. If not for the jolly music and overwhelming clamor of the crowd surrounding your little huddle, all would be hushed. "And that's when nine green-striped angels punched outta' hyperspace and began streaking towards our little drama. See, some of the local militia decided that justice was more important than jurisdiction, and they decid ...
```
```suggestion
			`	The listeners go quiet. Tense anticipation fills the air. If not for the jolly music and overwhelming clamor of the crowd surrounding your little huddle, all would be hushed. "And that's when nine green-striped angels punched outta' hyperspace and began streaking towards our little drama. See, some of the local militia decided that justice was more important than jurisdiction, and they decided t'give the smugglers an unasked-for escort. These fine folk arrived late to the party, after the shooting had already started, but just in time to make it stop. Syndicated Security's surrounded, and..." People huddled around the speaker are grinning eagerly. They've heard this story a hundred times - and evidently this is their favorite part. "The bastards stand down! Our smugglers didn't make any profit on accounta' having to turn back, and one pig out of the three hundred got rendered into bacon by laser fire, but folks got emboldened enough to start disregardin' contracts and began organizing their own shipping. Refused to use those crooks that were extortin' us as being the only way to get goods off planet. And that's why every year, on the anniversary of the Great Pig War, we celebrate."`
```

Amazinite commented on data/human/culture free worlds.txt:
```diff
@@ -0,0 +1,67 @@
+				decline
+
+mission "ZZCU South Culture: Repossessed"
+	landing
+	to offer
+		and
+			random < 2 + 50 * "Culture: Repossessed"
+			has "event: war begins"
+	source
+		government "Free Worlds"
+		attributes "spaceport" "farming" "south"
```
```suggestion
	minor
	to offer
		and
			random < 2 + 50 * "Culture: Repossessed"
			has "event: war begins"
	source
		government "Free Worlds"
		attributes "farming" "south"
```

Amazinite commented on data/human/culture free worlds.txt:
```diff
@@ -0,0 +1,67 @@
+		government "Free Worlds"
+		attributes "spaceport" "farming" "south"
+	on offer
+		conversation
+			`You arrive at the spaceport just in time to witness a shuttle being towed off its landing pad by a tractor, a familiar blazon displayed upon the spacecraft's doors. Though it's made somewhat harder to recognize by the fact that somebody has spray-painted "REPOSESED" over the logo, it appears that this shuttle belongs to - or used to belong to - the very same galactic bank that cut you the loa ...
+				decline
+	on decline
+		"Culture: Repossessed" ++
```
```suggestion
		"Culture: Repossessed" ++
		conversation
			`You arrive at the spaceport just in time to witness a shuttle being towed off its landing pad by a tractor, a familiar blazon displayed upon the spacecraft's doors. Though it's made somewhat harder to recognize by the fact that somebody has spray-painted "REPOSESED" over the logo, it appears that this shuttle belongs to - or used to belong to - the very same galactic bank that cut you the loan for your first ship. If the crowd of jeering locals is anything to go by, it looks like popular opinion here doesn't favor the galaxy's financial institution. A group of militia members observe the blatant ship-theft, evidently unwilling to intercede. In fact, some of them are cheering on the hijackers.`
				decline
```

Amazinite commented on data/human/culture free worlds.txt:
```diff
@@ -0,0 +1,67 @@
+	source
+		government "Free Worlds"
+		attributes "spaceport" "urban" "rim"
+	on offer
+		conversation
+			`As you exit your ship, you run overhear a spaceport technician attempting to have an animated conversation with another. "Piracy? And the militia does nothing? That goes against everything we stand for." Stern disapproval colors the speaker's voice. "These clowns're assaulting bankers and taking their shuttles, last I heard. I get that they're frustrated out there on the farms and owe money,  ...
+				decline
+	on decline
+		"Culture: Repossessed" ++
```
```suggestion
	on offer
		"Culture: Repossessed" ++
		conversation
			`As you exit your ship, you run overhear a spaceport technician attempting to have an animated conversation with another. "Piracy? And the militia does nothing? That goes against everything we stand for." Stern disapproval colors the speaker's voice. "These clowns're assaulting bankers and taking their shuttles, last I heard. I get that they're frustrated out there on the farms and owe money, but that doesn't excuse theft." The technician's partner, currently shoulder-deep in an engine, responds distractedly. "Cool. You gonna' pass me that buffer or not? This Port and Polish ain't about to finish itself." The pair continue to converse as you carry on your way, but it's out of earshot before long.`
				decline
```

Amazinite commented on data/human/culture free worlds.txt:
```diff
@@ -0,0 +1,67 @@
+		"Culture: Repossessed" ++
+		
+mission "ZZCU Rim Culture: Repossessed"
+	landing
+	to offer
+		and
+			random < 2 + 50 * "Culture: Repossessed"
+			has "event: war begins"
+	source
+		government "Free Worlds"
+		attributes "spaceport" "urban" "rim"
```
```suggestion
	minor
	to offer
		and
			random < 2 + 50 * "Culture: Repossessed"
			has "event: war begins"
	source
		government "Free Worlds"
		attributes "urban" "rim"
```

Amazinite commented on data/human/culture free worlds.txt:
```diff
@@ -0,0 +1,67 @@
+		attributes "spaceport" "urban" "rim"
+	on offer
+		conversation
+			`As you exit your ship, you run overhear a spaceport technician attempting to have an animated conversation with another. "Piracy? And the militia does nothing? That goes against everything we stand for." Stern disapproval colors the speaker's voice. "These clowns're assaulting bankers and taking their shuttles, last I heard. I get that they're frustrated out there on the farms and owe money,  ...
```
```suggestion
			`As you exit your ship, you can overhear a spaceport technician attempting to have an animated conversation with another. "Piracy? And the militia does nothing? That goes against everything we stand for." Stern disapproval colors the speaker's voice. "These clowns 'er assaulting bankers and taking their shuttles, last I heard. I get that they're frustrated out there on the farms and owe money, but that doesn't excuse theft." The technician's partner, currently shoulder-deep in an engine, responds distractedly. "Cool. You gonna pass me that buffer or not? This port and polish ain't about to finish itself." The pair continue to converse as you carry on your way, but it's out of earshot before long.`
```

Amazinite commented on data/human/culture republic.txt:
```diff
@@ -0,0 +1,125 @@
+#      Republic with Syndicate Pairing      #
+#############################################
+
+mission "ZZCU Sad Gamblers -R"
```
You basically have this same thing in two different regions. Maybe have some difference between the regions? I'd also suggest changing the names. Actually type out "Republic" and "Syndicate" instead of "-R" and "-S".

MasterOfGrey replied:
I was trying to get it to appear anywhere in either region but I couldn't get the source criteria to work right.

Amazinite commented on data/human/culture republic.txt:
```diff
@@ -0,0 +1,125 @@
+			`	You roll your eyes and put it back. Some experiences really are universal.`
+				decline
+			label buy
+			`	You purchase the mug on an impulse. At least you can ditch your other one with a chip in it.`
```
Let's give the player a mug outfit and take 50 credits from them by making this an invisible mission where `on accept` you get the outfit and lose the credits, then automatically fail the mission.

"Mug: Hand to hand outfit. +0.1 boarding attack. You can throw it in a pinch, you guess." - @Galaucus

Amazinite commented on data/human/culture republic.txt:
```diff
@@ -0,0 +1,125 @@
+		random == 1
+	on offer
+		conversation
+			`As you move about the spaceport you can't avoid the smell. It's like a trash fire, but nothing more than the smell is in the air.`
```
Took me like five tries to figure out what this was trying to say.
```suggestion
			`As you move about the spaceport you can't avoid the terrible smell. It's like a trash fire, but you can't see any sort of smoke.`
```

Amazinite commented on data/human/culture republic.txt:
```diff
@@ -0,0 +1,125 @@
+				`	(Ignore it.)`
+					goto ignore
+				`	(Ask around.)`
+			`	You ask one of the locals who doesn't look too busy. They sadly inform you that there's been a bumper banana crop, but unfortunately other, wealthier, worlds had a bumper crop at the same time, and there's nowhere to either sell or store a significant excess. Distressed farmers are burning the excess fruit to make fertilizer for the next season, the wind has carried the smell all over town.`
```
```suggestion
			`	You ask one of the locals who doesn't look too busy. They sadly inform you that there's been a bumper banana crop, but unfortunately other wealthier worlds had a bumper crop at the same time, and there's nowhere to either sell or store a significant excess. Distressed farmers are burning the excess fruit to make fertilizer for the next season, the wind has carried the smell all over town.`
```

Amazinite commented on data/human/culture republic.txt:
```diff
@@ -0,0 +1,125 @@
+		random == 1
+	on offer
+		conversation
+			`Visiting the Paradise spaceports is always an interesting experience for someone from abroad. Nowhere else is class such an ingrained component of daily life. You catch yourself noticing people who seem to be invisible to most of those going about their business, but they never catch your eye. Perhaps it is because of your background, but you see them even when they are trying to avoid notice ...
```
This last sentence reads strange to me. Maybe just ditch it.

Amazinite commented on data/human/culture independents.txt:
```diff
@@ -0,0 +1,68 @@
+		random == 1
+		has "main plot complete"
+	on offer
+		conversation
```
I'll have you know that this is a single-space-after-periods repository. I'm not going to create suggestions for every single line, but that needs changed.

Amazinite commented on data/human/culture independents.txt:
```diff
@@ -0,0 +1,68 @@
+			choice
+				`	(Take a closer look.)`
+					goto investigate
+				`	(Shout well wishes as you pass by)`
+					goto passby
+				`	(Ignore them, you have more important things to do.)`
```
```suggestion
				`	(Shout well wishes as I pass by)`
					goto passby
				`	(Ignore them, I have more important things to do.)`
```

Amazinite commented on data/human/culture independents.txt:
```diff
@@ -0,0 +1,68 @@
+			`	You hurry past the group as they quiet down.  Taking advantage of the momentary lull, you shout "Good luck!  You've got this!" towards them.  They let out another cheer in response, with the exception of the young woman who buries her face in her hands.`
+				decline
+			label investigate
+			`	You approach the group as they quiet down, noticing hint of anxiety in the air.`
```
```suggestion
			`	You approach the group as they quiet down, noticing a hint of anxiety in the air.`
```

Amazinite commented on data/human/culture independents.txt:
```diff
@@ -0,0 +1,68 @@
+			label investigate
+			`	You approach the group as they quiet down, noticing hint of anxiety in the air.`
+			`	"Nothing quite like buying your very first starship," you say over their chatter.  The celebration goes quiet as they turn to see who interrupted their little party.`
+			`	"It's certainly been an...experience."  The young woman replies once you are in comfortable talking distance.`
```
```suggestion
			`	"It's certainly been an... experience,"  the young woman replies once you are in comfortable talking distance.`
```

Amazinite commented on data/human/culture independents.txt:
```diff
@@ -0,0 +1,68 @@
+			`	"Nothing quite like buying your very first starship," you say over their chatter.  The celebration goes quiet as they turn to see who interrupted their little party.`
+			`	"It's certainly been an...experience."  The young woman replies once you are in comfortable talking distance.`
+			choice
+				`	"Have you got any big plans with it?"`
+					goto plans
+				`	(Give some advice.)`
+					goto advice
+				`	(Make a bad joke.)`
```
I try to avoid mixing "action" and "dialog" choices, especially when the action choices are just to say some dialog. You should take the dialog that follows these two action choices and just have them be the choices themselves.

Amazinite commented on data/human/culture independents.txt:
```diff
@@ -0,0 +1,68 @@
+				`	(Give some advice.)`
+					goto advice
+				`	(Make a bad joke.)`
+			`	"So, who's the lucky pilot?"  You ask, gesturing to the Shuttle.`
+			`	"Uhh...that would be me."  The woman says, glancing around awkwardly.`
```
```suggestion
			`	"So, who's the lucky pilot?" you ask, gesturing to the Shuttle.`
			`	"Uhh... that would be me," the woman says, glancing around awkwardly.`
```

Amazinite commented on data/human/culture independents.txt:
```diff
@@ -0,0 +1,68 @@
+			`	"Uhh...that would be me."  The woman says, glancing around awkwardly.`
+			choice
+				`	(Try to salvage the awkward situation.)`
+					goto salvage
```
Unnecessary goto/label.
```suggestion
```

Amazinite commented on data/human/culture independents.txt:
```diff
@@ -0,0 +1,68 @@
+					goto salvage
+				`	(Get out while you can.)`
+					goto getout
+			label salvage
```
```suggestion
```

Amazinite commented on data/human/culture independents.txt:
```diff
@@ -0,0 +1,68 @@
+					goto getout
+			label salvage
+			`	"I'm just messing with you, my psychiatrist doesn't think it's very funny when I do it to him either."`
+			`	"Oh," the woman says quietly, "that's...nice, I guess."`
+				goto getout
```
```suggestion
			`	"Oh," the woman says quietly, "that's... nice, I guess."`
```

Amazinite commented on data/human/culture independents.txt:
```diff
@@ -0,0 +1,68 @@
+			`	"Right now," she says with a sigh, "the only plan I have is to get rid of the massive debt that I'm in because of it."`
+				goto advice
+			label advice
+			`	"The mortgage isn't so bad as long as you're willing to work hard," you say with a smile, "and there is no shortage of people in need of a ship and a good pilot.`
```
```suggestion
			`	"The mortgage isn't so bad as long as you're willing to work hard," you say with a smile, "and there is no shortage of people in need of a ship and a good pilot."`
```

W1zrad commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+		random < 1
+	on offer
+		conversation
+			`While stopped in the spaceport waiting for refueling you are surprised by how hard it is to find somewhere to sit and have a meal & a drink. It seems Hai have crammed themselves into every bar or similar venue with a public display screen, arrayed in sashes and mantles and all manner of other paraphernalia which, near as you can tell, looks like... team sport merchandise?`
```
```suggestion
			`While stopped in the spaceport waiting for refueling, you are surprised by how hard it is to find somewhere to sit & have a meal and a drink. It seems Hai have crammed themselves into every bar or similar venue with a public display screen, arrayed in sashes and mantles and all manner of other paraphernalia which, near as you can tell, look like... team sport merchandise?`
```
```suggestion
			`While stopped in the spaceport waiting for refueling you are surprised by how hard it is to find somewhere to sit and have a meal and a drink. It seems Hai have crammed themselves into every bar or similar venue with a public display screen, arrayed in sashes and mantles and all manner of other paraphernalia which, near as you can tell, look like... team sport merchandise?`
```
Don't remember ever seeing `&` in-game. Was it there just to differentiate it from the `and` right before?

MasterOfGrey replied:
It was yes.

W1zrad commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+	on offer
+		conversation
+			`While stopped in the spaceport waiting for refueling you are surprised by how hard it is to find somewhere to sit and have a meal & a drink. It seems Hai have crammed themselves into every bar or similar venue with a public display screen, arrayed in sashes and mantles and all manner of other paraphernalia which, near as you can tell, looks like... team sport merchandise?`
+			`	You're not really an expert on exactly what such things look like in Hai society and so when you finally find somewhere to sit, at a bar for lack of any other options, you ask the tired looking bartender what exactly is going on.`
```
```suggestion
			`	You're not really an expert on exactly what such things look like in Hai society and so when you finally find somewhere to sit, at a bar for lack of any other options, you ask the tired-looking bartender what exactly is going on.`
```
`tired-looking` forms a single adjective describing `bartender`, so needs a dash. There's also two `exactly`s here. You could switch one of them out for variety if you wanted. Maybe a `precisely`? Or just take one out? Or nothing at all; your choice.

MasterOfGrey replied:
I think I'd take out the first one.

W1zrad replied:
```suggestion
			`	You're not really an expert on what such things look like in Hai society and so when you finally find somewhere to sit, at a bar for lack of any other options, you ask the tired looking bartender what exactly is going on.`
```

W1zrad commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+			`	You're not really an expert on exactly what such things look like in Hai society and so when you finally find somewhere to sit, at a bar for lack of any other options, you ask the tired looking bartender what exactly is going on.`
+			`	"Oh this? They're all here for the Musical Entertainment Expo streams. Unfortunately, none of them start drinking until at least one of their teams has announced something of worth," he says with an exasperated eye-roll.`
+			choice
+				`	"Sure, can I just get some food here?"`
```
The `Sure` feels a little out of place to me because the First Last isn't really affirming anything.
```suggestion
				`	"Okay. Can I just get some food here?"`
```
There's a lot of other alternatives you could do too.

W1zrad commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+			choice
+				`	"Sure, can I just get some food here?"`
+					goto wrapup
+				`	"Whats the Musical Entertainment Expo?"`
```
```suggestion
				`	"What's the Musical Entertainment Expo?"`
```

W1zrad commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+					goto wrapup
+				`	"Whats the Musical Entertainment Expo?"`
+				`	"Why don't they start drinking yet?"`
+			`	"Well you see, the big music labels do a lot of talent scouting all year round, they all have a department that is solely dedicated to finding and assembling new music groups to be the next big thing. It's tradition to celebrate only after you've committed to the journey with a new band. Every three years the industry holds ME2, which is what you're seeing up there, to debut their next round ...
```
```suggestion
			`	"Well you see, the big music labels do a lot of talent scouting all year round. They all have departments solely dedicated to finding and assembling new music groups to be the next big thing. It's tradition to celebrate only after you've committed to the journey with a new band. Every three years the industry holds ME2, which is what you're seeing up there, to debut their next round of bands and to tease any mid-season releases they might have in the works."`
```
I prefer the period for the longer pause it provides, and I think the other edit is just a little smoother. Again, totally up to you.

W1zrad commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+				`	"Do you not have musicians trying to make it on their own?"` 
+			`	"Oh for sure, but mostly just to get picked up by one of these companies."`
+			label production
+			`	The bartender chuckles. "We've been doing pop music for longer than your people have had fire, it's about the experience of being a fan of a group as it rises to fame and stardom. A dedicated Hai can go back and find a song for almost any possible life experience in almost any possible genre. It's the journey, and the personalities, that are what it's all about."`
```
```suggestion
			`	The bartender chuckles. "We've been doing pop music for longer than your people have had fire. it's about the experience of being a fan of a group as it rises to fame and stardom. A dedicated Hai can go back and find a song for almost any possible life experience in almost any possible genre. It's the journey, and the personalities, that are what it's all about."`
```
Same as above with the pause.

MasterOfGrey replied:
Subsequent capitalisation got missed. :P

W1zrad replied:
My bad!
```suggestion
			`	The bartender chuckles. "We've been doing pop music for longer than your people have had fire. It's about the experience of being a fan of a group as it rises to fame and stardom. A dedicated Hai can go back and find a song for almost any possible life experience in almost any possible genre. It's the journey, and the personalities, that are what it's all about."`
```

W1zrad commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+				`	"How interesting."`
+			`	"Anyway, I gather you were after some food?"`
+			label wrapup
+			`	The bartender tips his head in your direction and takes your order. "No problem, it'll be right up."`
+			`	It's a noisy meal.`
```
```suggestion
			`	The bartender tips his head in your direction and takes your order. "No problem, it'll be right up." It's a noisy meal.`
```
These two paragraphs are especially tiny. Stitch 'em together if ya want.

MasterOfGrey replied:
This separation is for emphasis.

W1zrad replied:
Fair enough. Feel free to resolve.

W1zrad commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+				has "language: Wanderer"
+			`	As you watch, the triangular elder being holoprojected seems to flicker and then fizzles out as the battery unit fades, apparently under the assault of the root vegetable which appears to do a victory dance, before proceeding to act out something that makes precisely no sense to you.`
+			`	A little further on you encounter two different groups of protesters weaving in and out of the spaceport's entrances like some kind of conga line. The triangular elder and opposing root vegetable make several reappearances on their placards, some of them holographically animated.`
+			`	A different security guard, this one in regular attire, appears at your elbow. "You look a little lost but don't worry, it's just a meme war. Carry on with your business and don't touch the meme items, no one should bother you over it."`
```
```suggestion
			`	A different security guard, this one in regular attire, appears at your elbow. "You look a little lost. Don't worry, it's just a meme war. Carry on with your business and don't touch the meme items, and no one should bother you over it."`
```
Optional; thought it helped with flow.

W1zrad commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+			label translator
+			`	As you watch, the triangular elder being holoprojected seems to flicker and then fizzles out as the battery unit fades, apparently under the assault of the root vegetable which appears to do a victory dance. To your surprise your translator pipes up, "[Victory, Success] for the true and square root that is more [significant, meaningful] than the tripartite [system, tradition] of the elders!" ...
+			`	A little further on you encounter two different groups of protesters weaving in and out of the spaceport's entrances like some kind of conga line. The triangular elder and opposing root vegetable make several reappearances on their placards, some of them holographically animated.`
+			`	A different security guard, this one in regular attire, appears at your elbow. "You look a little lost but don't worry, it's just a meme war. Carry on with your business and don't touch the meme items, no one should bother you over it."`
```
```suggestion
			`	A different security guard, this one in regular attire, appears at your elbow. "You look a little lost. Don't worry, it's just a meme war. Carry on with your business and don't touch the meme items, and no one should bother you over it."`
```
Copied+pasted from above.

W1zrad commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+				`	"Sure, will do."`
+					goto done
+				`	"Your elders have a tripartite system?"`
+			`	The security guard chuckles and says, "Oh you can understand? They don't really, but it's a very old joke from an old campaign that there are only three tenets the elders care about, and they're all the ones that affect us the least."`
```
Double old. Maybe `ancient` as an alternative, or remove one? Up to you.

MasterOfGrey replied:
```suggestion
			`	The security guard chuckles and says, "Oh you can understand? They don't really, but it's a very old joke from a bygone political campaign that there are only three tenets the elders care about, and they're all the ones that affect us the least."`
```

W1zrad commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+					goto done
+				`	"Your elders have a tripartite system?"`
+			`	The security guard chuckles and says, "Oh you can understand? They don't really, but it's a very old joke from an old campaign that there are only three tenets the elders care about, and they're all the ones that affect us the least."`
+			`	When you ask what has triggered this now he says, "Oh, the latest education priorities came out with a four tenet focus, on the same day that an elder accidentally quoted the old campaign. It has been very entertaining."`
```
```suggestion
			`	When you ask what has triggered this, he says, "Oh, the latest education priorities came out with a four tenet focus, on the same day that an elder accidentally quoted the old campaign. It has been very entertaining."`
```
I'd push for a comma here, but that's just me. The `now` seemed a little confusing. If we're asking what inciting event triggered the meme war, then I don't think it's necessary.

W1zrad reviewed (commented):
I think the most organizationally sound way to manage this is to note which missions I'm reviewing here, so here's what I did and some thoughts about them:

"ZZCU Theater Chat"
Maybe a little confusing to understand, but I think that's okay for a flavor conversation like this (see more elaboration at the bottom). Could use some paragraph combining where possible, but that's personal preference.

"ZZCU Musical Entertainment Expo"
Pseudo-E3 with sports getup but music focus? Acts as a really interesting of implying every species enjoys similar stuff, but we all celebrate it in different ways.

"ZZCU The Meme Wars 1"
Can't have a game made by the internet without talking about memes, this it is an absolute requirement that this get in

About the missions/PR overall:
I think this could act as a pretty cool way to give individual writers more stylistic/creative freedom with their writing. In these cases, there's very little concerns about style preventing less-advanced readers from understanding important plot points, so I'm all for it. As such, I'm going to try to keep the stylistic side of my reviewing to a minimum and mostly focus on grammar/spelling checks (even with those I'll try to be a little lax to preserve rhythm/flow where it helps to have things that bend the rules). Looking forward to reviewing more of these when I have the chance!

MasterOfGrey commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+			`As you are walking through the spaceport, you come across the uncommon pairing of a Hai and a human, their conversation standing out by virtue of speaking the common tongue of humanity.`
+			`	"Oh, oh, Gemanari, this looks interesting!" The human woman looks to be in her late sixties and, when you turn to look, seems to be clucking over a holographic advertisement for a theater show. "This looks adorable, doesn't it? Oh, and it says they're using ripple ray refractive holographic flight effects!"`
+			`	The advertisement does indeed look very impressive and you pause to observe for a moment.`
+			`	The Hai, presumably Gemanari, also appears to be a more mature lady who squints at it and wrinkles her nose. "Ugh, that's the Germontian interpretation though."`
```
*brain pain* what?

W1zrad replied:
Whoops. That sentence has some strange phrasing, so I thought it was wrong and subsequently made an error trying to fix it. You're good to leave it as-is then.

MasterOfGrey commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+			`	The advertisement does indeed look very impressive and you pause to observe for a moment.`
+			`	The Hai, presumably Gemanari, also appears to be a more mature lady who squints at it and wrinkles her nose. "Ugh, that's the Germontian interpretation though."`
+			`	This receives a puzzled look to which Gemanari thinks for a second before clarifying, "You would say the Social Darwinist interpretation."`
+			`	The lady grimaces and makes like she's vomiting, "I can't believe you still have those schools of thought in a society this old."`
```
I think I meant this more like a subtle mime than a look.

W1zrad replied:
Oh okay. First suggestion should be good then

W1zrad commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+				`	(Ignore them and pass by.)`
+					goto skip
+				`	(Take a look.)`
+			`	The crowd appears to be mostly made up of younger Hai though a few more mature Hai are also there, visibly exhibiting more restraint on their enthusiasm. Reaching the edge of the crowd and peering over the shorter Hai folk you can see a table set up with two standing banners flanking it and a stack of, at a glance, glossy printed photos.`
```
```suggestion
			`	The crowd appears to be mostly made up of younger Hai, though a few more mature Hai are also there, visibly exhibiting more restraint on their enthusiasm. Reaching the edge of the crowd and peering over the shorter Hai folk, you can see a table set up with two standing banners flanking it and a stack of, at a glance, glossy printed photos.`
```

W1zrad commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+				`	(Ignore them and pass by.)`
+					goto skip
+				`	(Take a look.)`
+			`	The crowd appears to be mostly made up of younger Hai though a few more mature Hai are also there, visibly exhibiting more restraint on their enthusiasm. Reaching the edge of the crowd and peering over the shorter Hai folk you can see a table set up with two standing banners flanking it and a stack of, at a glance, glossy printed photos.`
```
```suggestion
			`	The crowd appears to be mostly made up of younger Hai, though a few more mature Hai are also there, visibly exhibiting more restraint on their enthusiasm. Reaching the edge of the crowd and peering over the shorter Hai folk, you can see a table set up with two standing banners flanking it and a stack of, at a glance, glossy printed photos.`
```

W1zrad commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+					goto skip
+				`	(Take a look.)`
+			`	The crowd appears to be mostly made up of younger Hai though a few more mature Hai are also there, visibly exhibiting more restraint on their enthusiasm. Reaching the edge of the crowd and peering over the shorter Hai folk you can see a table set up with two standing banners flanking it and a stack of, at a glance, glossy printed photos.`
+			`	The banners depict a svelte Hai in the strikingly familiar costume of a fencer, with a holographically enhanced animation that makes it appear to be continuously performing an elegant parry and riposte against an out-of-frame opponent. The effect is, though, achieved solely through a trick of artificially shifting perspective. To your surprise you realize that this is a signing event for a f ...
```
Ooo, `svelte`. Had to look that one up. Nice word choice there

W1zrad commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+				`	(Ignore them and pass by.)`
+					goto skip
+				`	(Take a look.)`
+			`	The crowd appears to be mostly made up of younger Hai though a few more mature Hai are also there, visibly exhibiting more restraint on their enthusiasm. Reaching the edge of the crowd and peering over the shorter Hai folk you can see a table set up with two standing banners flanking it and a stack of, at a glance, glossy printed photos.`
```
```suggestion
			`	The crowd appears to be mostly made up of younger Hai, though a few more mature Hai are also there, visibly exhibiting more restraint on their enthusiasm. Reaching the edge of the crowd and peering over the shorter Hai folk, you can see a table set up with two standing banners flanking it and a stack of, at a glance, glossy printed photos.`
```

W1zrad commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+					goto skip
+				`	(Take a look.)`
+			`	The crowd appears to be mostly made up of younger Hai though a few more mature Hai are also there, visibly exhibiting more restraint on their enthusiasm. Reaching the edge of the crowd and peering over the shorter Hai folk you can see a table set up with two standing banners flanking it and a stack of, at a glance, glossy printed photos.`
+			`	The banners depict a pair of Hai, one with a net and one with what looks like a bladed frisbee. It's not at all clear what precisely it is that they do and when the individuals in question emerge from a booth behind the setup and greet their fans it becomes clear that, if you could even make yourself heard over the trilling, no one is particularly keen to explain it to you. The pair remove t ...
```
```suggestion
			`	The banners depict a pair of Hai, one with a net and one with what looks like a bladed frisbee. It's not at all clear what precisely it is that they do, and when the individuals in question emerge from a booth behind the setup and greet their fans it becomes clear that, if you could even make yourself heard over the trilling, no one is particularly keen to explain it to you. The pair remove their helmets and flash dazzling Hai smiles at their fans before sitting down with some kind of pen for the assembled Hai to line up.`
```

W1zrad commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+				`	(Place an order.)`
+					goto end
+				`	"Hey, I noticed there are some dark looks around here. Are human captains not welcome?"`
+			`	The barkeep looks confused for a second before looking over at the table and shaking his head in understanding, "Oh no, you're perfectly welcome. That group is just breaking the maturity tenets, some Hai get uncomfortable with adults blatantly acting like children in public."`
```
```suggestion
			`	The barkeep looks confused for a second before looking over at the table and shaking his head in understanding. "Oh no, you're perfectly welcome. That group is just breaking the maturity tenets, some Hai get uncomfortable with adults blatantly acting like children in public."`
```

W1zrad commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+				`	"The maturity tenets?"`
+			`	"Oh right, I suppose you wouldn't know. There are a number of tenets that Hai are taught growing up, lessons to be internalized to be a responsible and functioning member of society. The second is 'To be honest in dealings,' and the third is 'To gamble only on skill, not chance,' so their game breaks two of them."`
+			choice
+				`	"Some would say Poker is more about skill than chance."`
```
```suggestion
				`	"Some would say poker is more about skill than chance."`
```

W1zrad commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+				`	"Some would say Poker is more about skill than chance."`
+				`	"Surely if it's just a game though, there's no harm in it?"`
+			`	"Oh no, I'm afraid the first tenet ensures that it's really not favorable either way. The first is 'To have clear separation of fantasy and reality,' so a game cannot have real consequences or it is reality, not fantasy. So to gamble money on the game is to bring real consequences to a fantasy exercise. This is taboo for an adult.`
+			`	"If you interpret the game as being played for the skill, without the wagering, then you clear the first tenet, but you have the issue that the principle skills are bluffing and manipulation, which violates the second tenet. Bluffing isn't a skill seen as having virtue, nor is manipulation."`
```
```suggestion
			`	"If you interpret the game as being played for the skill, without the wagering, then you clear the first and third tenet, but you have the issue that the principle skills are bluffing and manipulation, which violates the second tenet. Bluffing isn't a skill seen as having virtue, nor is manipulation."`
```
Skill passes 3rd, no wagering passes 1st, right?

W1zrad commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+			`	"If you interpret the game as being played for the skill, without the wagering, then you clear the first tenet, but you have the issue that the principle skills are bluffing and manipulation, which violates the second tenet. Bluffing isn't a skill seen as having virtue, nor is manipulation."`
+			choice
+				`	"Wait, do the Hai not lie or bluff?"`
+			`	The barkeep chortles with a strange trill and says, "Oh no the Hai do lie and bluff, we're actually very good at bluffing, evolutionarily advantageous in the wild and all that. It's just seen as very immature and base. An adult should be able to conduct their lives without resorting to such primitive instincts. We learned that the hard way long ago, back before we holed up in our current bor ...
```
```suggestion
			`	The barkeep chortles with a strange trill and says, "Oh no, the Hai do lie and bluff. We're actually very good at bluffing, evolutionarily advantageous in the wild and all that. It's just seen as very immature and base. An adult should be able to conduct their lives without resorting to such primitive instincts. We learned that the hard way long ago, back before we holed up in our current borders."`
```

W1zrad commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+				`	"Wait, do the Hai not lie or bluff?"`
+			`	The barkeep chortles with a strange trill and says, "Oh no the Hai do lie and bluff, we're actually very good at bluffing, evolutionarily advantageous in the wild and all that. It's just seen as very immature and base. An adult should be able to conduct their lives without resorting to such primitive instincts. We learned that the hard way long ago, back before we holed up in our current bor ...
+			`	He looks at you intently before continuing, "You won't hear the tenets brought up very much, but if you look closely you'll see that the first three especially are ingrained in our society very deeply."`
+			`	With a shrug he flips open his order pad and pats his belly. "Of course, my gran always said that any of them after the tenth were more guidelines than actual rules, and I certainly have my share of disagreements with thirteen. Now, before I talk your ears off, what'll I get for you?"`
```
`more guidelines than actual rules`
Is this a Pirates of the Caribbean reference? I hope it is

MasterOfGrey replied:
It is. ;)

W1zrad commented on data/hai/culture hai.txt:
```diff
@@ -0,0 +1,235 @@
+		has "language: Wanderer"
+	on offer
+		conversation
+			`A mall stall at the spaceport is trying to sell learning merchandise that's been condemned by the local conglomerate of mum groups. There's an argument and some flipped tables. (TBD)`
```
```suggestion
			`A mall stall at the spaceport is trying to sell learning merchandise that's been condemned by the local conglomerate of mom groups. There's an argument and some flipped tables. (TBD)`
```
Still yet to be written? Also - American spelling prefers `mom` over `mum`.

MasterOfGrey replied:
Yeah this is gonna get taken out in the clean-up and revisited

W1zrad reviewed (changes requested):
Here's what I reviewed this time:
"ZZCU Wrestling Star"
"ZZCU Fencing Star"
"ZZCU Mystery Sport Star"
The only concern I'd have is that the intro is identical for all of these, which may cause people to assume that they're reading the same thing over again and skip it before they realize each is unique. But since these come up so rarely, I doubt that will often be an issue.
"ZZCU Gambling Judgment"
The tenets are a pretty interesting concept.
"ZZCU Mum Groups"
"ZZCU boop"
Looks like these two are still a work in progress?

MasterOfGrey commented:
Danke @W1zrad 

`But since these come up so rarely, I doubt that will often be an issue.` That's pretty much what I'm relying on.

W1zrad commented on data/human/culture republic.txt:
```diff
@@ -0,0 +1,124 @@
+		conversation
+			`Visiting the Paradise spaceports is always an interesting experience for someone from abroad. Nowhere else is class such an ingrained component of daily life. You find yourself noticing people who seem to be invisible to most of those going about their business, but they never catch your eye. They always appear to be doing their utmost to avoid notice.`
+			`	You haven't noticed children that look poor here in the past, but on this occasion you spot one right before he knocks over a mop and bucket... right in front of a businessman and his family rolling their bags through the spaceport. You've never seen a mother go so deathly pale as the cleaner who scurried to fix things before the man had even drawn breath.`
+			`	A pair of security officers appear almost immediately, before you could intercede one way or the other, as the man and his family begin to complain. One officer provides assurances as the other supervises the cleaner, who quickly hurries her children out of harms way.`
```
```suggestion
			`	A pair of security officers appear almost immediately, before you could intercede one way or the other, as the man and his family begin to complain. One officer provides assurances as the other supervises the cleaner, who quickly hurries her child out of harms way.`
```
She isn't really mentioned as having more than one child before, so it feels a little weird to bring it up here. You could also keep this as-is and add some mention of her having additional kids earlier, but that might be more work.

W1zrad commented on data/human/culture syndicate.txt:
```diff
@@ -0,0 +1,57 @@
+		random < 1
+	on offer
+		conversation
+			`Passing through the spaceport you see a sizable group of people waiting on a lottery draw. You hear a whoop of success and your attention is drawn as one of them wins a moderate sum in a 12th division win; a day or two worth of standard pay.`
```
```suggestion
			`Passing through the spaceport you see a sizable group of people waiting on a lottery draw. You hear a whoop of success, and your attention is drawn as one of them wins a moderate sum in a 12th division win, a day or two worth of standard pay.`
```
OR
```suggestion
			`Passing through the spaceport you see a sizable group of people waiting on a lottery draw. You hear a whoop of success, and your attention is drawn as one of them wins a moderate sum in a 12th division win. It's a day or two worth of standard pay.`
```
OR
```suggestion
			`Passing through the spaceport you see a sizable group of people waiting on a lottery draw. You hear a whoop of success, and your attention is drawn as one of them wins a moderate sum in a 12th division win; it's a day or two worth of standard pay.`
```

W1zrad reviewed (changes requested):
Wrapped up everything else that didn't seem to be a work in progress. Quick question I've noticed: do the copyright year and name need to be updated?

MasterOfGrey commented on data/human/culture free worlds.txt:
```diff
@@ -0,0 +1,65 @@
+					defer
+				`	(Stop and take in the sights - and maybe a soft pretzel or three.)`
+			`	Conversation with the locals eventually turns to the matter of what they're celebrating in the first place. As it turns out, the bronzed pig holds the clue. Today is the anniversary of the Great Pig War, a conflict which apparently raged between the Southern Defense Pact and the Syndicate unbeknownst to the larger galaxy. Toasts are raised in honor of nine heroic captains - and to the memory ...
+			`	"Back in the forties," a reveler drunkenly explains to you, "half the farms 'n' ranches hones' folks about this planet were either about ready to leave or takin' out a second mortgage on their property. Bankers from richer worlds were always comin' about to repossess something or another." A passer-by spits on the ground in disgust at the mention - or perhaps he'd just succeeded in extricati ...
```
Honestly neither am I, this ones getting taken out and thrown back to Lake for the clean-up.

MasterOfGrey commented on data/human/culture republic.txt:
```diff
@@ -0,0 +1,124 @@
+		random < 1
+	on offer
+		conversation
+			`As you wander the spaceport with a snack, waiting for your ship to be refueled, you pass a local souvenir shop and poke your head in to kill a little time. You see a decent looking mug with the text "<source>" printed underneath an attractive, stylized representation of the spaceport you are presently standing in and pick it up to see how it feels. The handle is solid, and not too small for y ...
```
See in this case it's a souvenir, so the planet would seem logical to me.

MasterOfGrey commented:
To the best of my knowledge this is merge-ready.

MasterOfGrey commented:
Merged in Amazinite's PR'd changes. 

However, still would like a response actually addressing the issues from Amazinite:

For the file name we had actually chatted on discord and you seemed ok with Culture Conversations (vastly preferred by me) at the time.

For the prefixes you explained why you didn't think it was necessary, but never what it it was worth to you personally to explicitly take away the value that other people saw in it.

MasterOfGrey commented:
Alright, well I would still like to argue for "culture conversations" because I would also like to include in it some conversations that trigger silently and then have situations like, you overhearing ghost stories on the comms in certain areas and stuff. So I do see it as more restrictive, especially when the overwhelming majority of the file is going to be highlighting cultural aspects of the regions they offer in.

Thankyou for the full answer on the other point though. That's valid.

Amazinite commented:
> I would also like to include in it some conversations that trigger silently and then have situations like, you overhearing ghost stories on the comms in certain areas and stuff.

Well we've already noted earlier how "transport missions" has outgrown its original name and now contains missions other than transport related ones. It's notable though that none of the other factions contain such a "transport missions" file, with the responsibility for that file being taken up by `<faction> missions` files. (For example. "hai missions" contains missions that if they were only in human space would be in the "transport missions" file.) So how about renaming "transport missions" to "generic missions" or "human missions" (less inclined to use the latter option)? Then anything that doesn't fit elsewhere would more clearly just be placed in that file.

MasterOfGrey commented:
In that case I definitely maintain then that they're actually conversations for the purpose of giving the player more insight into the culture of the area they're in, that typically but don't necessarily occur in the spaceport. Meaning I would rather put the in-space convos and other things I have up my sleeve here, and let non-cultural conversations go to "human missions", than the reverse. 

Thematically the purpose of the file is definitely stronger with culture conversations. Especially now you've renamed that file to cover more generic stuff.

Amazinite commented on data/hai/hai culture conversations.txt:
```diff
@@ -0,0 +1,209 @@
+		random < 1
+	on offer
+		conversation
+			`Stepping into the spaceport you are beset by a riot of color. Every blank surface that's not usually paid advertising space seems to be festooned with dangling holographic projectors, banners and ribbons. There's even actual paper fliers and posters, apparently taped to whatever surface will bear them. At first you wonder if there's some sort of festival going on, but as you watch a young Hai ...
```
```suggestion
			`Stepping into the spaceport you are beset by a riot of color. Every blank surface that's not usually paid advertising space seems to be festooned with dangling holographic projectors, banners, and ribbons. There's even actual paper fliers and posters, apparently taped to whatever surface will bear them. At first you wonder if there's some sort of festival going on, but as you watch a young Hai wearing a hood slaps up another holo projector, creating a cartoonish root vegetable with a sword, which begins hacking at the adjacent hologram, some kind of impressionist concept of a Hai elder made up of iridescent triangles. The young Hai then proceeds to tear down a banner and slap some goo on a massive poster which begins dissolving before a pair of security guards, wearing some very obviously non-standard additions to their uniforms, chase him back out of the spaceport.`
```

Amazinite commented on data/hai/hai culture conversations.txt:
```diff
@@ -0,0 +1,209 @@
+			branch translator
+				has "language: Wanderer"
+			`	As you watch, the triangular elder being holoprojected seems to flicker and then fizzles out as the battery unit fades, apparently under the assault of the root vegetable which appears to do a victory dance, before proceeding to act out something that makes precisely no sense to you.`
+			`	A little further on you encounter two different groups of protesters weaving in and out of the spaceport's entrances like some kind of conga line. The triangular elder and opposing root vegetable make several reappearances on their placards, some of them holographically animated.`
```
```suggestion
			`	A little further on you encounter two different groups of protesters, assuming that's what they are, weaving in and out of the spaceport's entrances like some kind of conga line. The triangular elder and opposing root vegetable make several reappearances on their placards, some of them holographically animated.`
```

Amazinite commented on data/hai/hai culture conversations.txt:
```diff
@@ -0,0 +1,209 @@
+				has "language: Wanderer"
+			`	As you watch, the triangular elder being holoprojected seems to flicker and then fizzles out as the battery unit fades, apparently under the assault of the root vegetable which appears to do a victory dance, before proceeding to act out something that makes precisely no sense to you.`
+			`	A little further on you encounter two different groups of protesters weaving in and out of the spaceport's entrances like some kind of conga line. The triangular elder and opposing root vegetable make several reappearances on their placards, some of them holographically animated.`
+			`	A different security guard, this one in regular attire, appears at your elbow. "You look a little lost. Don't worry, it's just a meme war. Carry on with your business and don't touch the meme items, and no one should bother you over it."`
```
```suggestion
			`	A different security guard, this one in regular attire, appears at your elbow. "You look a little lost. Don't worry, it's just a meme war. Carry on with your business and don't touch the meme items and no one should bother you over it."`
```

Amazinite commented on data/hai/hai culture conversations.txt:
```diff
@@ -0,0 +1,209 @@
+			`	You thank the security guard for his advice and go about what you were doing as the baffling conflict continues around you.`
+				decline
+			label translator
+			`	As you watch, the triangular elder being holoprojected seems to flicker and then fizzles out as the battery unit fades, apparently under the assault of the root vegetable which appears to do a victory dance. To your surprise your translator pipes up, "[Victory, Success] for the true and square root that is more [significant, meaningful] than the tripartite [system, tradition] of the elders!" ...
```
```suggestion
			`	As you watch, the triangular elder being holoprojected seems to flicker and then fizzles out as the battery unit fades, apparently under the assault of the root vegetable which appears to do a victory dance. To your surprise, your translator pipes up: "[Victory, Success] for the true and square root that is more [significant, meaningful] than the tripartite [system, tradition] of the elders!"`
```

Amazinite commented on data/hai/hai culture conversations.txt:
```diff
@@ -0,0 +1,209 @@
+				decline
+			label translator
+			`	As you watch, the triangular elder being holoprojected seems to flicker and then fizzles out as the battery unit fades, apparently under the assault of the root vegetable which appears to do a victory dance. To your surprise your translator pipes up, "[Victory, Success] for the true and square root that is more [significant, meaningful] than the tripartite [system, tradition] of the elders!" ...
+			`	A little further on you encounter two different groups of protesters weaving in and out of the spaceport's entrances like some kind of conga line. The triangular elder and opposing root vegetable make several reappearances on their placards, some of them holographically animated.`
```
```suggestion
			`	A little further on you encounter two different groups of protesters, assuming that's what they are, weaving in and out of the spaceport's entrances like some kind of conga line. The triangular elder and opposing root vegetable make several reappearances on their placards, some of them holographically animated.`
```

Amazinite reviewed (changes requested):
My final review.
